src/dotnet/Program.cs

```diff
@@ -36,6 +37,7 @@ public class Program
             ["build"] = BuildCommand.Run,
             ["clean"] = CleanCommand.Run,
             ["help"] = HelpCommand.Run,
+            ["list"] = (new ListCommand()).Run,
```

Comment:
We missed this in previous PR's @krwq. If you want to change the invocation pattern for commands then we need to do it wholesale. These three instances have put us in a place where it's not clear what pattern to use when adding a new command. Unless there is a reason to make a change globally, we should be consistent with the original pattern for all three of these.